components/cookie-consent.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+    <v-snackbar
+      v-model="show"
+      transition="fade"
+      :vertical="true"
+      :timeout="10000000"
+      :app="true"
+      :absolute="true"
+      content-class="snack-content"
+      :right="true"
+      outlined
+      rounded
+    >
+            <span
+              class="cookie-consent-message"
+            >
+          {{ $t('cookie.message') }}
+
+
+        </span>
+
+      <template #action="{ }">
+
+      <v-row no-gutters>
+        <v-col>
+          <small-button-cheese
+            class="ma-2"
+            @click="leave"
+          >
+            {{ $t('cookie.refuse') }}
+          </small-button-cheese>
+        </v-col>
+        <v-col>
+
+          <small-button-cheese
+            class="ma-2"
+            @click="dismiss"
+          >
+            {{ $t('cookie.accept') }}
+          </small-button-cheese>
+        </v-col>
+      </v-row>
+
+
+      </template>
+    </v-snackbar>
+</template>
+
+<script>
+export default {
+  name: 'CookieConsent',
+  props: {
+    // default
+    transition: {
+      type: String,
+      default: 'cookie-consent-transition'
+    }
+
+  },
+  data() {
+    return {
+      show: undefined
+    }
+  },
+  computed: {
+    cookie() {
+      return !this.getCookie('cookie_a_la_raclette')
+    }
+  },
+  beforeMount() {
```

Comment:
Pourquoi pas juste dans le `mounted` ou `created` ?

Reply:
car j'ai copier coller le code d'un gadjo mais apres plus serieuse recherche

Server-side rendering (Nuxt, Vuepress etc.)
docs
beforeCreate and created are only hooks called on the server
that means you should not use any code which needs window, document or any browser API in created as those will not be present on the server
on the other hand code placed in the beforeMount (or mounted) is executed only on the client (browser)